src/github-release.ts

```diff
@@ -88,11 +90,13 @@ export default class GitHubRelease {
 
   constructor(
     options: Partial<IGitHubOptions>,
+    hooks: IAutoHooks,
```

Comment:
why wouldn't `GitHubRelease` make it's own hooks?

Reply:
Good point!